versions/latest/modules/en/pages/networking/basic_network_options.adoc

```diff
@@ -95,12 +95,16 @@ metadata:
   namespace: kube-system
 spec:
   valuesContent: |-
-    eni:
+    encryption:
       enabled: true
```

Comment:
```suggestion
      enabled: true
      type: wireguard
```

This change is missing: https://github.com/rancher/rke2-docs/pull/403/files#diff-19f62af259130623f20cfdc126843c9703afb930939269864f176d0609f4507cR90